registry.json

```diff
@@ -15,6 +15,22 @@
     "operatorName": "Jack Salamone",
     "github": "jacksalamone"
   },
+  {
+    "ticker": "CNFT",
+    "poolId": "68cc6dc773c067328a43e5e21086b515d23f8326c59035245996f34b",
+    "social": {
+      "website": "CNFTpool.com",
+      "discord": "",
+      "reddit": "",
+      "instagram": "",
+      "facebook": "",
+      "youtube": "https://www.youtube.com/channel/UCpz7w-c5CNqEyp6XW1YMkcA",
+      "twitter": "LordForAll"
+    },
+    "telegram": "",
```

Comment:
The alliance rules are that you must supply the TG handle of an individual (not group or channel)

Reply:
Has been updated now thanks